RELEASE.md

```diff
@@ -5,6 +5,12 @@ at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Prepare for Source Code Release
 
+> If this release considers a Neo version update (e.g. `3.4.0` &rarr; `3.5.0`): Check whether the [Neo-Express
```

Comment:
maybe., add it as a "major" Neo version update?

Reply:
yes, added in e79a774